### PR TITLE
DESK-930 Announcements

### PR DESCRIPTION
**Pull request by JamieRuderman**
### Description

Provide in app messaging

### Test plan

1. Sign in
2. See announcements indicator
3. Click it and see announcements
4. You should see the indicator disappear after reading.

### Ticket

<https://remoteit.atlassian.net/browse/DESK-930>

### CODE REVIEW

File: frontend/src/models/announcements.ts [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+      return state
+    },
+  },
+})
+
+export function selectAnnouncements(state: ApplicationState, unread?: boolean) {
+  return state.announcements.all.filter(a => !unread || !a.read)
+}
+
+const TEST_DATA: any = [
```
Comment (mauricio0000) [inline frontend/src/models/announcements.ts:81]: do you need this TEST_DATA?

File: frontend/src/pages/AnnouncementsPage.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+import { AnnouncementCard } from '../components/AnnouncementCard'
+import { Container } from '../components/Container'
+import { Title } from '../components/Title'
+import { Icon } from '../components/Icon'
+
+export const AnnouncementsPage = () => {
+  const enabled = useSelector((state: ApplicationState) => selectAnnouncements(state))
+
+  return (
+    <Container
```
Comment (mauricio0000) [inline frontend/src/pages/AnnouncementsPage.tsx:15]: are you to show this component if `enabled` doesn't have data?

File: frontend/src/models/announcements.ts (hunk 1 above)
Comment (JamieRuderman) [reply frontend/src/models/announcements.ts:81]: Don't need, but it's useful for testing.

File: frontend/src/pages/AnnouncementsPage.tsx (hunk 2 above)
Comment (JamieRuderman) [reply frontend/src/pages/AnnouncementsPage.tsx:15]: Yes we would show the page anyway. It just would be empty until the announcements are loaded. I rename enabled to annoucements. It used to be that I was checking enabled, but Benoit said he's checking that cloud side so I removed it.

**Pull request merged**